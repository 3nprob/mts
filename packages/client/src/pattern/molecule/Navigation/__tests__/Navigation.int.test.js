import React from 'react'
import { defaultRender } from './Navigation.int.render'

jest.mock('react-router-dom', () => ({ Link: () => <div>some link</div> }))

describe('<Navigation />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
