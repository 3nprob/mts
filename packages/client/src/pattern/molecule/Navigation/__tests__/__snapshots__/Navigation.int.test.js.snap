// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> Snaphot must match defaultRender 1`] = `
<nav
  className="nav"
>
  <div
    className="navBar"
  >
    <a
      className="navBarLogo"
      href="/"
    />
    <ul
      className="navBarList"
    >
      <li
        className="navBarListItem"
      >
        <a
          className="navBarListItemLink"
          href="/about"
        >
          About
        </a>
      </li>
      <li
        className="navBarListItem"
      >
        <a
          className="navBarListItemLink"
          href="/blog"
        >
          Blog
        </a>
      </li>
      <li
        className="navBarListItem"
      >
        <a
          className="navBarListItemLink"
          href="/courses"
        >
          Courses
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
